config.sh

```diff
@@ -37,7 +38,9 @@ function build_osx_wheel {
     # Build dual arch wheel
     export CC=clang
     export CXX=clang++
-    brew install pkg-config
+    command -v pkg-config >/dev/null 2>&1 || {
```

Comment:
With latest multibuild, you can now do `install_pkg_config` - it just runs that command above, but at least it puts the logic in one place where it can be maintained.